.github/workflows/tests-api.yml

```diff
@@ -157,7 +157,7 @@ jobs:
             alembic upgrade pre@head
             alembic upgrade post@head
             flask install_data
-            python tests/alembic/check_db_schema.py
+#            python tests/alembic/check_db_schema.py
```

Comment:
Avec un petit FIXME pour expliquer pourquoi on le commente et pour penser à le remettre quand c'est corrigé ? Proposition : 

    FIXME: this command fails since dbb3f7e8ec16ca66af88a0320ea83aae61d0cc8e on GitHub Action (but
    not on CircleCI). It blocks deployment on testing. Restore it when the command is fixed.

Reply:
Malheureusement, cela échoue avant, lorsque la 1e migration est jouée.
Je désactive toute le job, on l'a (bien écrit) sur CircleCI.